src/components/Home/MetricsCards.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo, useRef } from 'react'
+import Flickity from 'react-flickity-component'
+
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import useResponsive from 'decentraland-gatsby/dist/hooks/useResponsive'
+import { Container } from 'decentraland-ui/dist/components/Container/Container'
+import Responsive from 'semantic-ui-react/dist/commonjs/addons/Responsive'
+
+import { DclData } from '../../api/DclData'
+import { ProposalStatus } from '../../entities/Proposal/types'
+import useProposals from '../../hooks/useProposals'
+
+import MetricsCard from './MetricsCard'
+import './MetricsCards.css'
+
+const flickityOptions = {
+  cellAlign: 'left',
+  accessibility: true,
+  pageDots: false,
+  prevNextButtons: false,
+  draggable: true,
+  dragThreshold: 10,
+  selectedAttraction: 0.01,
+  friction: 0.15,
+}
+
+const MetricsCards = () => {
+  const responsive = useResponsive()
```

Comment:
we should stop using useResponsive as it's deprecated, there's a new hook (refactor fridays!)